src/modules/github/github.actions.js

```diff
@@ -0,0 +1,18 @@
+import types from "./github.types";
+import GitHubAPI from "./github.api";
+
+const api = new GitHubAPI();
```

Comment:
Pamiętaj, że możemy też używać funkcji w pliku `github.api.js` i je exportować ;)
To oczywiście nie błąd - tak tylko pisze informacyjnie ;)